hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/TransactionScenarioBuilder.java

```diff
@@ -138,6 +139,8 @@ public static TransactionBody goodDefaultBody() {
                         .build())
                 .transactionFee(1L)
                 .transactionValidDuration(Duration.newBuilder().seconds(60).build())
+                .nodeAccountID(ALICE.accountID())
```

Comment:
We should use a different `AccountID`, IMO. Otherwise, the payer and the node account are the same, which is not typical.